vcf/vcfAncestor_test.go

```diff
@@ -68,10 +68,11 @@ func TestVcfAnnotateAncestorFromFa(t *testing.T) {
 	reader, _ := GoReadToChan("testdata/TestVcfAnnotateAncestorFromFa.vcf")
 	records := fasta.Read("testdata/testAncestorSequence.fa")
 	var i int = 0
+	var currRef, currAln int = 0, 0
```

Comment:
Do we need something inside the loop that sets these back to zero when we go to a new chromosome?